### PR TITLE
COMMERCE-11353 - Price Range Facet is not working

### PR DESCRIPTION
**Pull request by victorcdp**
Related Issue: [COMMERCE-11353](https://issues.liferay.com/browse/COMMERCE-11353)

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java [hunk 1]
```diff
@@ -837,6 +837,10 @@ else if (!cpInstances.isEmpty()) {
 			}
 
 			for (CPInstance cpInstance : cpInstances) {
+				if (cpInstance.isExpired()) {
+					continue;
+				}
```
Comment (brianikim) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java:842]: Did you check the indexing for Control Panel -> Products (All/Skus) view? The expired products/skus should still be shown in there.

Comment (brianikim) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java:842]: To confirm, once you deploy the change here, rerun the indexer, in Control Panel-> Search -> Index Actions -> Reindex Search Indexes.

**Comment by brianikim** [comment]
forward it to liferay-commerce and assign it to Andrea Sbarra.

**Pull request closed without merging**